app/api/v1/endpoints/tenants.py

```
@@ -159,6 +167,106 @@ async def update_tenant(
+        raise HTTPException(status_code=404, detail="Tenant not found")
+    if tenant.integrity_profile == body.new_profile:
+        latest = await history_repo.get_latest_for_tenant(tenant_id)
+        return TenantIntegrityStatus(
+            profile=tenant.integrity_profile,
+            last_changed_at=latest.changed_at if latest else None,
+            cooling_off_ends_at=latest.cooling_off_ends_at if latest else None,
+        )
+
+    # For now, changes are effective immediately; effective_from_seq is set to 0.
+    previous_profile = tenant.integrity_profile
+    updated = await tenant_repo.update_tenant(
```

Flake8 (7.3.0)

[error] local variable 'updated' is assigned to but never used

(F841)